assets/styles/pages/_standard-attributes.scss

```diff
@@ -0,0 +1,18 @@
+.standard-attributes-main{
+
+    #standard-attributes-nav {
+        position: sticky;
+        top: 95px;
+        align-self: flex-start;
+        z-index: 2;
+        background-color: white;
+        padding-top: 12px;
+        padding-bottom: 12px;
+
+        .search-standard-attributes input {
+            padding: 9px 8px;
+            border-radius: 4px;
+            border: #C7C7C7 1px solid 
```

Comment:
can we re-use the scss variable here?  `$gray-light` should be, i think this is missing the closing semi colon as well

Reply:
thanks for the recommendation! 👍